mne/io/reference.py

```diff
@@ -197,6 +197,9 @@ def add_reference_channels(inst, ref_channels, copy=True):
     else:  # n_ref_channels == 1 and a single ref digitization exists
         ref_dig_array = np.concatenate((ref_dig_loc[0]['r'],
                                        ref_dig_loc[0]['r'], np.zeros(6)))
+        # Replace the (possibly new) Ref location for each channel
```

Comment:
That is true about CAR and might be part (or all?) of the reason that we require it for inverse modeling, but no projections (including CAR) get used during forward computation IIRC. That's the best I can do for explanation, if you're curious feel free to ping Matti -- he might enjoy explaining it.